api/src/test/java/org/openmrs/aop/AuthorizationAdviceTest.java

```diff
@@ -63,8 +63,8 @@ public void before_shouldNotifyListenersAboutCheckedPrivileges() {
 		
 		Context.getConceptService().saveConcept(concept);
 		
-		assertThat("listener1", listener1.hasPrivileges, containsInAnyOrder("Manage Concepts", "Get Observations", "Get Concept Attribute Types"));
-		assertThat("listener2", listener2.hasPrivileges, containsInAnyOrder("Manage Concepts", "Get Observations", "Get Concept Attribute Types"));
+		assertThat("listener1", listener1.hasPrivileges, containsInAnyOrder("Manage Concepts", "Get Observations", "Get Concept Attribute Types", "Get Concept Map Types"));
```

Comment:
is your question related to this specific diff? it confuses me since codacy doesnt complain about this part  ;)

a contributor with merge privileges can ignore any of the PR checks and simply merge. however comments by codacy should help contributors adhere to our style and prevent error prone code from getting merged.

I just checked codacy's comments and will try to add exceptions to test classes for MagicNumbers and throwing Exception since that affects all tests